src/JuMP.jl

```diff
@@ -738,5 +729,19 @@ include("print.jl")
 # Nonlinear-specific code
 include("nlp.jl")
 
+
+typealias _getTypes Union{NonlinearExpression,NonlinearParameter, AffExpr}
+getValue{T<:_getTypes}(arr::Array{T}) = map(getValue, arr)
```

Comment:
Having ``NonlinearExpression <: JuMPTypes`` would define a lot of operator overloads which don't necessarily make sense. We could do ``Union{JuMPTypes, NonlinearExpression}`` though.